README.md

```diff
@@ -6,6 +6,7 @@ The deployment scripts are separated into versions.
 
 - v1.0.1
 - v1.1.0
+- v1.2.0
```

Comment:
The versions section is starting to get pretty big; how about we move it to the bottom of the file and put it under a "Changelog" heading?